build.gradle

```diff
@@ -1361,6 +1375,21 @@ project(':clients') {
     generator project(':generator')
   }
 
+  shadowJar {
+    archiveClassifier = null
+    // KIP-714: move shaded dependencies to a shaded location
+    relocate('io.opentelemetry', 'org.apache.kafka.shaded.io.opentelemetry')
```

Comment:
can we limit the relocation to the more specific `io.opentelemetry.proto` package since there shouldn't be any other opentelemetry classes.

Reply:
Done.